src/analyzer.rs

```diff
@@ -427,6 +427,9 @@ fn array_recurse_for_type(
         match items {
             JSONSchemaPropsOrArray::Schema(s) => {
                 if s.type_.is_none() && s.x_kubernetes_preserve_unknown_fields == Some(true) {
+                    if s.x_kubernetes_preserve_unknown_fields == Some(true) {
+                        return Ok(("Vec<serde_json::Value>".into(), level));
+                    }
                     let map_type = cfg.map.name();
                     return Ok((format!("Vec<{}<String, serde_json::Value>>", map_type), level));
```

Comment:
wonder why i wrapped this in a map earlier. curious to see if it breaks any other tests.

Reply:
although, that is a bit weird.. the test `array_of_preserve_unknown_objects` should be doing the same thing

Reply:
ah.. of course. that is the test you are changing.